### PR TITLE
feat: Add a new list of random messages to display after refresh

### PR DESCRIPTION
**Pull request by DreuXXX**
@orhun  I have added the new messages for my issue, sorry for taking it too long from my side. I would like to contribute more. We can discuss moving these const arrays into database.

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -139,6 +139,33 @@ const SHARE_MESSAGES = [
   "OMG!",
 ];
 
+const RANDOM_MESSAGES = [
```
Comment (orhun) [inline utils.js:142]: Are these messages only showed on page refresh? The point of #40 is to show random but related messages when you refresh the page. I'm not sure how is that possible though.

Comment (orhun) [reply utils.js:142]: It seems like these messages are shown either way, not depending on the page refresh.

Comment (DreuXXX) [reply utils.js:142]: Oh, sorry, I kinda understanded the issue some other way. I will correct it today or tomorrow and make a commit ASAP.

Comment (DreuXXX) [reply utils.js:142]: Alright, I have done it. I think. You can check it now.

Comment (orhun) [reply utils.js:142]: Can we rename `RANDOM_MESSAGES` to `REFRESH_MESSAGES` to make it more explanatory?

File: pages/index.js [hunk 2]
```diff
@@ -40,11 +41,20 @@ function Index({ trackId, backSideContent, latestId }) {
...
+      localStorage.setItem("refresh", false)
+      setRandomMsg(getRandomFromArray(RANDOM_MESSAGES))
+    } 
     ReactGA.initialize("UA-172416216-1");
     ReactGA.pageview(window.location.pathname + window.location.search);
     document.getElementById(
       "app"
     ).style.backgroundImage = `url('/images/${getRandomFromArray(BG_IMAGES)}')`;
+    window.onbeforeunload = function () {
+      console.log("reload")
```
Comment (orhun) [inline pages/index.js:54]: Please remove this line.

File: pages/index.js [hunk 3]
```diff
@@ -40,11 +41,20 @@ function Index({ trackId, backSideContent, latestId }) {
...
+      localStorage.setItem("refresh", true)
+      return true;
```
Comment (orhun) [inline pages/index.js:56]: I think we need to remove this return to get rid of the dialog which appears when you try to refresh the page:

cap

A quick google search led me to this page: https://stackoverflow.com/questions/17188058/using-onbeforeunload-without-dialog

**Review by orhun — CHANGES_REQUESTED** [review]
Looks good! Just a few changes needed 🐻

File: pages/index.js (hunk 3 above)
Comment (DreuXXX) [reply pages/index.js]: Fixed. Hopefully :)

**Review by orhun — APPROVED** [review]
Great stuff!

**Pull request merged**